darwin/exporter/formats/nifti.py

```diff
@@ -228,95 +274,152 @@ def populate_output_volumes(
     ----------
     annotation : Union[dt.Annotation, dt.VideoAnnotation]
         The Union of these two files used to populate the volume with
-    output_dir : Path
-        The folder where the new instance mask files will be.
     slot_map : Dict
         Dictionary of the different slots within the annotation file
     output_volumes : Dict
         volumes created from the build_output_volumes file
-    image_id : str
 
     Returns
     -------
     volume : dict
-        Returns the populated volume
+        Returns dict of volumes with class names as keys and volumes as values
+
+    """
+    for annotation in annotations:
+        slot_name = annotation.slot_names[0]
+        slot = slot_map[slot_name]
+        series_instance_uid = slot.metadata.get(
+            "SeriesInstanceUID", "SeriesIntanceUIDNotProvided"
+        )
+        volume = output_volumes.get(series_instance_uid)
+        frames = annotation.frames
+
+        # define the different planes
+        XYPLANE = 0
```

Comment:
Are these intrinsinc to nifti and potentially used by other functions? If so can we make this a type

Something like
```python
class Plane(Enum):
    XY = 0
    XZ = 1
    YZ = 2

if view_idx == Plane.XY.value
```

Reply:
Implemented in the new commit